gridstack/gridstack.d.ts

```diff
@@ -1,241 +1,347 @@
-﻿// Type definitions for Gridstack
+﻿// Type definitions for Gridstack v0.2.5
 // Project: http://troolee.github.io/gridstack.js/
-// Definitions by: Pascal Senn <https://github.com/PascalSenn/>
```

Comment:
please keep original author name. add your name below.
like this https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/jquery/jquery.d.ts#L3